certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/certificatemodel/model/PdfQuestionField.java

```diff
@@ -0,0 +1,13 @@
+package se.inera.intyg.certificateservice.domain.certificatemodel.model;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class PdfQuestionField {
+
+  ElementId questionId;
+  String pdfFieldId;
```

Comment:
Should we use the type FieldId here instead? We try to work with objects if possible in the domain layer!

Reply:
Changing this to PdfFieldId as discussed since they are two different things.